### PR TITLE
Add userGroupAdded and userGroupRemoved to discord webhook formatter

### PR DESCRIPTION
**Pull request by DominikSLK**
I added userGroupAdded and userGroupRemoved events to discord webhook formatter and fixed hardcoded New Announcement.

### CODE REVIEW

File: core/classes/Core/User.php [hunk 1]
```diff
@@ -181,11 +181,14 @@ public function addGroup(int $group_id, int $expire = 0, $group_data = null): bo
             $this->_groups[$group_id] = new Group($group_data);
         }
 
+        $default_language = new Language('core', DEFAULT_LANGUAGE);
         EventHandler::executeEvent('userGroupAdded', [
             'username' => $this->data()->username,
             'user_id' => $this->data()->id,
             'group_id' => $group_id,
             'group_name' => $this->_groups[$group_id]->name,
+            'avatar_url' => AvatarSource::getAvatarFromUserData($this->data()),
```
Comment (partydragen) [inline core/classes/Core/User.php:190]: You are missing the full url param, anyways you should use `$user->getAvatar(128, true),` instead

File: core/classes/Core/User.php [hunk 2]
```diff
@@ -708,11 +711,14 @@ public function removeGroup(?int $group_id): bool {
             ]
         );
 
+        $default_language = new Language('core', DEFAULT_LANGUAGE);
         EventHandler::executeEvent('userGroupRemoved', [
             'username' => $this->data()->username,
             'user_id' => $this->data()->id,
             'group_id' => $group_id,
             'group_name' => $this->_groups[$group_id]->name,
+            'avatar_url' => AvatarSource::getAvatarFromUserData($this->data()),
```
Comment (partydragen) [inline core/classes/Core/User.php:720]: same here as above

**Review by partydragen — CHANGES_REQUESTED** [review]

**Review by partydragen — APPROVED** [review]

**Comment by DominikSLK** [comment]
🤔

**Review by samerton — APPROVED** [review]

**Comment by samerton** [comment]
Thanks!

**Pull request merged**